sktime/classification/deep_learning/rnn.py

```diff
@@ -63,10 +63,19 @@ def __init__(
         activation="sigmoid",
         use_bias=True,
         optimizer=None,
+        n_epochs=100,
     ):
         _check_dl_dependencies(severity="error")
         super().__init__()
-        self.num_epochs = num_epochs
+        # Deprecated Parameter Handling
+        if num_epochs is not None:
+            warnings.warn(
```

Comment:
minor thing, we should use `warn` from `sktime.utils.warnings`, as that is affected by estimator level configs.